.github/workflows/foundry.yml

```diff
@@ -27,3 +27,6 @@ jobs:
 
       - name: Run snapshot
         run: forge snapshot --match-contract gas
+
+      - name: Run format
+        run: forge fmt
```

Comment:
not ideal to run forge fmt in actions. Since forge fmt will change the files. Best to do forge fmt check and if it doesn't work `forge fmt --check` and it should fail the whole action